### PR TITLE
Profiler

### PR DESCRIPTION
**Pull request by piotr-j**
Some fixes. Silly git why is that in there...

### CODE REVIEW

File: contrib-plugin-profile/pom.xml [hunk 1]
```diff
@@ -30,20 +31,18 @@
...
             <artifactId>gdx</artifactId>
-            <version>1.6.4</version>
+            <version>${libgdx.version}</version>
         </dependency>
         <dependency>
             <groupId>com.badlogicgames.gdx</groupId>
             <artifactId>gdx-backend-lwjgl</artifactId>
-            <version>1.6.4</version>
-            <scope>test</scope>
+            <version>${libgdx.version}</version>
```
Comment (antag99) [inline contrib-plugin-profile/pom.xml]: Shouldn't these dependencies be `<scope>test</scope>`?

Comment (piotr-j) [inline contrib-plugin-profile/pom.xml]: Yeah it should

Comment (DaanVanYperen) [inline contrib-plugin-profile/pom.xml]: > Shouldn't these dependencies be <scope>test</scope>?

removed by accident!

:cookie: thanks for PR!

Comment (antag99) [inline contrib-plugin-profile/pom.xml]: It was @DaanVanYperen!

**Comment by piotr-j** [comment]
I guess you can make a System for Plugin from the one in test if you really want, tho need to include the skin files as well.

**Pull request merged**